lib/ansible/modules/network/netvisor/pn_snmp_community.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_snmp_community
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.8"
+short_description: CLI command to create/modify/delete snmp-community
+description:
+  - This module can be used to create SNMP communities for SNMPv1 or
+    delete SNMP communities for SNMPv1 or modify SNMP communities for SNMPv1.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: false
+  state:
+    description:
+      - State the action to perform. Use C(present) to create snmp-community and
+        C(absent) to delete snmp-community C(update) to update snmp-community.
+    required: true
+    type: str
+    choices: ['present', 'absent', 'update']
+  pn_community_type:
+    description:
+      - community type.
+    type: str
+    choices: ['read-only', 'read-write']
+  pn_community_string:
+    description:
+      - community name.
+    type: str
+"""
+
+EXAMPLES = """
+- name: snmp-community functionality
+  pn_snmp_community:
+    pn_cliswitch: "sw01"
+    state: "present"
+    pn_community_string: "foo"
+    pn_community_type: "read-write"
+
+- name: snmp-community functionality
```

Comment:
@rajaspachipulusu17 can u please update the name accordingly to what the play actually does as the current one have generic name.